build.js

```diff
@@ -1,6 +1,12 @@
 const StyleDictionary = require("style-dictionary").extend("config.json");
 const ChangeCase = require('change-case')
 
+function isStringPxValue(token) {
```

Comment:
this folder is getting cluttered up with these functions/transforms, may be worth pulling out into its own file